templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -253,11 +247,9 @@ export class {{ service.name }}Client {
 {%- if api.legacyProtoLoad %}
 
     let protoFilesRoot = new this._gaxModule.GoogleProtoFilesRoot();
-{%- for filename in api.filesToGenerate %}
```

Comment:
One question, why do we need to remove the filesToGenerate condition? We only has one proto to generate?

Reply:
Yes, the idea is to have just one proto here (the one that defines the service).  If we find it does not work for some use cases, we'll think how to fix it.